apps/condo/pages/ticket/index.tsx

```diff
@@ -893,7 +893,7 @@ export const TicketTypeFilterSwitch = ({ ticketFilterQuery }) => {
             ...ticketFilterQuery,
         },
     })
-    // NOTE: we have index "ticket_assignee_exec_org" for this filter.
+    // NOTE: we have index "ticket_org_assignee_exec" for this filter.
```

Comment:
```suggestion
    // NOTE: we have index "ticket_org_assignee_exec_deletedAt" for this filter.
```

Reply:
thank you, fixed and made the name shorter (it was too long for migrator)